pom.xml

```diff
@@ -78,7 +78,7 @@
     </scm>
 
     <properties>
-        <geotools.version>20.1</geotools.version>
+        <geotools.version>21.2</geotools.version>
```

Comment:
I see Geotools 21 was the first version with Java 11 compatibility. I assume we aren't jumping to an even newer Geotools because of the transitive dependency on JTS, which has changed its package name and will conflict with other OTP dependencies?

Reply:
I tought that 21.2 _is_ the newest stable version: https://github.com/geotools/geotools/tags

Or would you like to use 22-RC?

Reply:
Oh I suppose it is. I saw 23-SNAPSHOT and must have assumed there was already a 22 release. But now I see 22 is also still in SNAPSHOT. In that case then there's no question, 21.2 is fine.